include/camp/camp.hpp

```diff
@@ -154,6 +155,37 @@ namespace test
 }
 #endif
 
+template<typename T, typename L>
+struct index_of;
+template<typename T, typename ...Elements>
+struct index_of<T, list<Elements...>> {
+  template<typename Seq, typename Item>
```

Comment:
Is there a better name we could use here instead of 'Seq', or is there no possibility that this could be confused with 'sequential'?

Reply:
The visibility of Seq is only within the template — it could be expanded to Sequence, but it looks readable to me as-is.

Reply:
Works for me. 👍